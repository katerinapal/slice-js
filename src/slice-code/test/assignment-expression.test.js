import {runAllCombosTests} from './helpers/utils'

/*
test.only(
  'orAndOr(false, false, true, true)',
  require('./helpers/utils').snapSlice(
    require.resolve('./fixtures/logical-expressions'),
    ({orAndOr}) => {
      return [orAndOr(false, false, true, true)]
    },
  ),
)
/* */

runAllCombosTests({
  filename: require.resolve('./fixtures/assignment-expressions'),
  methods: [
    {
      methodName: 'plusEqual',
      explicitArgs: [[15, 5]],
    },
    {
      methodName: 'minusEqual',
      explicitArgs: [[15, 5]],
    },
    {
      methodName: 'divideEqual',
      explicitArgs: [[15, 5]],
    },
    {
      methodName: 'multiplyEqual',
      explicitArgs: [[15, 5]],
    },
  ],
})
