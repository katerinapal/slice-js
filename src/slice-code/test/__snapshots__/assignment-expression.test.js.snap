// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`divideEqual(15, 5) 1`] = `
"export { divideEqual };

function divideEqual(thingOne, thingTwo) {
  thingOne /= thingTwo;
  return thingOne;
}"
`;

exports[`minusEqual(15, 5) 1`] = `
"export { minusEqual };

function minusEqual(thingOne, thingTwo) {
  thingOne -= thingTwo;
  return thingOne;
}"
`;

exports[`multiplyEqual(15, 5) 1`] = `
"export { multiplyEqual };

function multiplyEqual(thingOne, thingTwo) {
  thingOne *= thingTwo;
  return thingOne;
}"
`;

exports[`plusEqual(15, 5) 1`] = `
"export { plusEqual };

function plusEqual(thingOne, thingTwo) {
  thingOne += thingTwo;
  return thingOne;
}"
`;
